### PR TITLE
feat: update ordhook schemas

### PR DESCRIPTION
**Pull request by lgalabru**

### CODE REVIEW

File: components/chainhook-sdk/src/observer/mod.rs [hunk 1]
```diff
@@ -136,6 +136,67 @@ impl EventObserverConfig {
...
+                .and_then(|c| c.bitcoind_rpc_username.clone())
+                .unwrap_or("devnet".to_string()),
+            bitcoind_rpc_password: overrides
+                .and_then(|c| c.bitcoind_rpc_password.clone())
+                .unwrap_or("devnet".to_string()),
+            bitcoind_rpc_url: overrides
+                .and_then(|c| c.bitcoind_rpc_url.clone())
+                .unwrap_or("http://localhost:18443".to_string()),
+            bitcoin_block_signaling: overrides
+                .and_then(|c| match c.bitcoind_zmq_url.as_ref() {
```
Comment (hugocaillard) [inline components/chainhook-sdk/src/observer/mod.rs:173]: `.and_then` could be a `.map`
(and avoid the `Some(` L174 and 175)

File: components/chainhook-sdk/src/observer/mod.rs [hunk 2]
```diff
@@ -136,6 +136,67 @@ impl EventObserverConfig {
...
+                        ),
+                    )),
+                })
+                .unwrap_or(BitcoinBlockSignaling::Stacks(
+                    StacksNodeConfig::default_localhost(
+                        overrides
+                            .and_then(|c| c.ingestion_port)
+                            .unwrap_or(DEFAULT_INGESTION_PORT),
+                    ),
+                )),
```
Comment (hugocaillard) [inline components/chainhook-sdk/src/observer/mod.rs:189]: I think the duplication in the `None` branch and the `unwrap_or` can be avoid

```rust
            bitcoin_block_signaling: overrides
                .and_then(|c| c.bitcoind_zmq_url.as_ref())
                .map(|url| BitcoinBlockSignaling::ZeroMQ(url.clone()))
                .unwrap_or(BitcoinBlockSignaling::Stacks(
                    StacksNodeConfig::default_localhost(
                        overrides
                            .and_then(|c| c.ingestion_port)
                            .unwrap_or(DEFAULT_INGESTION_PORT),
                    ),
                )),
```

I _think_ it's the same behaviour

**Review by hugocaillard — COMMENTED** [review]
Suggested a refactor that could provide a small readability improvement

File: components/chainhook-sdk/src/observer/mod.rs (hunk 2 above)
Comment (lgalabru) [reply components/chainhook-sdk/src/observer/mod.rs]: Good idea, thanks!

Comment (lgalabru) [reply components/chainhook-sdk/src/observer/mod.rs]: Addressed with [latest](https://github.com/hirosystems/chainhook/pull/458/commits/687e2ae7b367f3c1ec173e5c56b471945622540d)

**Review by hugocaillard — APPROVED** [review]
LGTM 👍

**Pull request merged**